do-dont/doc/do-dont.md

```diff
@@ -0,0 +1,125 @@
+```js script
+import '@divriots/dockit-core/do-dont/dockit-do-dont.define.js';
+```
+
+# dockit-do & dockit-dont
+
+Component to render general instructions or for a specific component.
+It can take a component in the "component" slot, and instructions html in the "instructions" slot.
+Label, color and icon can be customised.
+See setup and usage examples below:
+
+## Setup
+
+```js
+import '@divriots/dockit-core/do-dont/dockit-do-dont.define.js';
+```
+
+## Demos
+
+### Do
+
+```html preview-story
+<dockit-do>
+  <ul slot="instructions">
+    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
+    <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
+    <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
+  </ul>
+</dockit-do>
+```
+
+#### Do with component
+
+```html preview-story
+<dockit-do>
+  <div
+    slot="component"
+    style="background-color: #00000030; padding: .5rem; font-weight: 600; border-radius: .25rem;"
+  >
+    SAMPLE BUTTON
+  </div>
+  <ul slot="instructions">
+    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
+    <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
+    <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
+  </ul>
+</dockit-do>
+```
+
+### Don't
+
+```html preview-story
+<dockit-dont>
+  <ul slot="instructions">
+    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
+    <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
+    <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
+  </ul>
+</dockit-dont>
+```
+
+### Don't with component
+
+```html preview-story
```

Comment:
you can use `js preview-story` and reuse `button` var defined above in the templates as you also do in stories file
any reasons you decided to write stories differently here?

Reply:
No reason, I just didn't know about this capabilities. Thanks for pointing out, fixed.